PSI_Tracker.js

```diff
@@ -0,0 +1,681 @@
+            " | ";
+        }
+      }
+
+      // STEP 3. We build the new row based on what we read on the Results first line
+      if (api === "PSI" || api === "PSI_A11Y_S12Y") {
+        const sheet_results = RESULT_SHEET;
+        const results_header = RESULT_SHEET_HEADER;
+        // We translate the dic output to an array row based on indexes found in "Results"
+        let array = [];
+        // Go through all the Dimensions listed in the header and build the array
+        for (let j = 0; j < results_header[0].length; j++) {
+          let dimension = results_header[0][j];
+          if (dimension in output) {
+            array.push(output[dimension]);
+          } else {
+            // We ultimately decided to leave blank columns without data and leave the details in the debugging.
+            // array.push("No data")
+            array.push("");
+            debug +=
+              "💢 Error on " + dimension.toString() + " - No data" + " | ";
+          }
+        }
+        // Append the array at the end of "Results"
+        sheet_results.appendRow(array);
+      }
+      if (api === "CrUX") {
+        const sheet_results = RESULT_SHEET;
+        const results_header = RESULT_SHEET_HEADER;
+        // We translate the dic output to an array row based on indexes found in "Results"
+        let array = [];
+        // Go through all the Dimensions listed in the header and build the array
+        for (let j = 0; j < results_header[0].length; j++) {
+          let dimension = results_header[0][j];
+          if (dimension in output) {
+            array.push(output[dimension]);
+          } else {
+            array.push("");
+            // Not debugging this
+            // debug += "💢 Error on " + dimension.toString() + " - No data" + " | "
+          }
+        }
+        // Append the array at the end of "Results"
+        sheet_results.appendRow(array);
+      }
+      if (api === "CrUX History") {
+        if ("Date" in output) {
+          for (let entry = 0; entry < output["Date"].length; entry++) {
+            const sheet_results = RESULT_SHEET;
+            const results_header = RESULT_SHEET_HEADER;
+            // We translate the dic output to an array row based on indexes found in "Results"
+            let array = [];
+            // Go through all the Dimensions listed in the header and build the array
+            for (let j = 0; j < results_header[0].length; j++) {
+              let dimension = results_header[0][j];
+              if (
+                ["Label", "URL", "Device", "URL / Origin", "CrUX URL"].indexOf(
+                  dimension
+                ) != -1 &&
+                dimension in output
+              ) {
+                array.push(output[dimension]);
+              } else if (dimension in output) {
+                array.push(output[dimension][entry]);
+              } else {
+                array.push("");
+                // Not debugging this
```

Comment:
Why isn't this specific case being debugged?

Reply:
Here it was meant as "not adding this error to the Debug tab"